### PR TITLE
fix(SSO):saved SSO intergration test login failed

### PR DESCRIPTION
**Pull request by ungreat**
#### What type of PR is this?
saved SSO intergration test login failed

#### What this PR does / why we need it:
cause by [issue](https://github.com/oceanbase/odc/pull/99)
Encryption#secret is set to  Access.WRITE_ONLY

#### Which issue(s) this PR fixes:

Fixes https://github.com/oceanbase/odc/issues/674
fix can't test  a saved SSO intergration login 

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/integration/oauth2/SSOEventHandler.java [hunk 1]
```diff
@@ -47,37 +51,43 @@ public boolean support(IntegrationEvent integrationEvent) {
...
 
     @Override
     public void preUpdate(IntegrationEvent integrationEvent) {
         Verify.notNull(addableClientRegistrationManager, "addableClientRegistrationManager");
+        IntegrationConfig preConfig = integrationEvent.getPreConfig();
         IntegrationConfig currentConfig = integrationEvent.getCurrentConfig();
         configurationValidator.checkNotEnabledInDbBeforeSave(currentConfig.getEnabled(),
                 currentConfig.getOrganizationId(), currentConfig.getId());
-        SSOIntegrationConfig ssoIntegrationConfig = getDecryptConfiguration(currentConfig);
+        // current config will not have secret when it is updated, secret can't change, so use preConfig
```
Comment (smallsheeeep) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/integration/oauth2/SSOEventHandler.java:75]: `currentConfig` may have `secret` if the secret is modified by user during update operation. So here may need a judgement ?

Comment (ungreat) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/integration/oauth2/SSOEventHandler.java:75]: secret is designed not to be updated

Comment (ungreat) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/integration/oauth2/SSOEventHandler.java:75]: secret is designed not to be updated

**Review by smallsheeeep — APPROVED** [review]
LGTM

**Pull request merged**